lib/post-json.js

```diff
@@ -22,23 +23,23 @@ function _performLocalDeployment(service, baseURL, what, callback) {
 
   var postURL = url.parse(baseURL);
   if (postURL.protocol !== 'http:') {
-    console.error('Invalid URL `%s`. Only http:// URLs are supported.',
+    glb.error('Invalid URL `%s`. Only http:// URLs are supported.',
       baseURL
     );
-    return callback(Error('Invalid URL'));
+    return callback(glb.Error('Invalid URL'));
   }
 
   if (postURL.hostname) {
     if (postURL.hostname !== 'localhost' && postURL.hostname !== '127.0.0.1') {
-      console.error('Invalid URL `%s`. Only localhost URLs are supported.',
+      glb.error('Invalid URL `%s`. Only {{localhost}} URLs are supported.',
         baseURL);
-      return callback(Error('Invalid URL'));
+      return callback(glb.Error('Invalid URL'));
```

Comment:
This looks really weird to me, since it appears to be changing the entire Error object, not just the message. Ignoring the lack of `new`, shouldn't it be something like this?
```js
return callback(Error(t('Invalid URL')));
```